src/controllers/ratingController.js

```diff
@@ -69,7 +77,44 @@ class ArticleRatingController {
     articleRatingStar = sum / articleRatings.length;
     return articleRatingStar.toFixed(1);
```

Comment:
@marcdomain nice implementation here 👍🏿, I just learnt something new from you using 'toFixed()'